src/libNetwork/P2PComm.cpp

```diff
@@ -691,6 +737,43 @@ void P2PComm::EventCallback(struct bufferevent* bev, short events,
   }
 }
 
+void P2PComm::EventCbServerSeed(struct bufferevent* bev, short events,
+                                [[gnu::unused]] void* ctx) {
+  LOG_MARKER();
+  int fd = bufferevent_getfd(bev);
+  struct sockaddr_in cli_addr {};
+  socklen_t addr_size = sizeof(struct sockaddr_in);
+  getpeername(fd, (struct sockaddr*)&cli_addr, &addr_size);
+  Peer peer(cli_addr.sin_addr.s_addr, cli_addr.sin_port);
+  LOG_GENERAL(DEBUG, "P2PSeed EventCbServer peer=" << peer << " bev=" << bev);
+
+  // TODO Remove all if conditions except last one. For now debugging purpose
```

Comment:
Maybe we can check the current log level and put all these flag checks inside `if level == DEBUG` (so it's easier to manage in the future if we need to do debugging)

Reply:
Enclosed all `flag`checks inside if statements.
`if (DEBUG_LEVEL == 4)`